crates/l2/README.md

```diff
@@ -0,0 +1,75 @@
+# Rust Ethereum L2
+
+## ToC
+
+- [How to run](#how-to-run)
+    - [The command you're looking for](#the-command-youre-looking-for)
+    - [The other command you will look for in the future](#the-other-command-you-will-look-for-in-the-future)
+    - [Other useful commands](#other-useful-commands)
+        - [General](#general)
+        - [L1](#l1)
+        - [L2](#l2)
+- [Local L1 Rich Wallets](#local-l1-rich-wallets)
+
+## How to run
+
+### The command you're looking for
+
+Running the below command will start both a local L1 (reth for the moment, but `ethereum_rust` in the future) and a local L2 (`ethereum_rust`).
+
+```
+make init
```

Comment:
This step needs forge, can we add how to install it?

Reply:
Sure!

Reply:
Added here [36552cd](https://github.com/lambdaclass/ethereum_rust/pull/570/commits/36552cd977aacc69a27497f80d78182b47785575)